<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoDataRiverPathPointExpandDialog</class>
 <widget class="QDialog" name="GeoDataRiverPathPointExpandDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extend Traversal Line</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="distanceRadioButton">
       <property name="text">
        <string>Set Distance between River Center and Left Bank</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="RealNumberEditWidget" name="distanceEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="incrementRadioButton">
       <property name="text">
        <string>Increment</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="RealNumberEditWidget" name="incrementEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="ratioRadioButton">
       <property name="text">
        <string>Extension Ratio</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <widget class="RealNumberEditWidget" name="ratioEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RealNumberEditWidget</class>
   <extends>QLineEdit</extends>
   <header>guibase/realnumbereditwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeoDataRiverPathPointExpandDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeoDataRiverPathPointExpandDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ratioRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>ratioEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>incrementRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>incrementEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>distanceRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>distanceEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
